options/options.go

```diff
@@ -43,6 +44,7 @@ type Options struct {
 	TLSCertFilePath          string   `envconfig:"KAFKA_TLS_CERT_FILE_PATH"`
 	TLSInsecureSkipTLSVerify bool     `envconfig:"KAFKA_TLS_INSECURE_SKIP_TLS_VERIFY" default:"true"`
 	TLSPassphrase            string   `envconfig:"KAFKA_TLS_PASSPHRASE"`
+	SASLMechanism            string   `envconfig:"KAFKA_SASL_MECHANISM" default:""`
```

Comment:
Can you move this up to the other SASL options so that these are grouped together?